src/pytest_cpp/google.py

```diff
@@ -72,12 +83,8 @@ def run_test(
         harness: Sequence[str] = (),
     ) -> tuple[list[GoogleTestFailure] | None, str]:
         with tempfile.TemporaryDirectory(prefix="pytest-cpp") as tmp_dir:
-            xml_filename = os.path.join(tmp_dir, "cpp-report.xml")
-            args = list(harness) + [
-                executable,
-                "--gtest_filter=" + test_id,
-                "--gtest_output=xml:%s" % xml_filename,
-            ]
+            xml_filename = os.path.join(os.path.relpath(tmp_dir), "cpp-report.xml")
```

Comment:
Same problem here regarding `relpath` and cwd.

Reply:
see my commect upper